CHANGELOG.md

```diff
@@ -47,6 +47,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 * (bank) [\#9618](https://github.com/cosmos/cosmos-sdk/pull/9618) Update bank.Metadata: add URI and URIHash attributes.
 * [\#9837](https://github.com/cosmos/cosmos-sdk/issues/9837) `--generate-only` flag will accept the keyname now.
 * [\#10045](https://github.com/cosmos/cosmos-sdk/pull/10045) Revert [#8549](https://github.com/cosmos/cosmos-sdk/pull/8549). Do not route grpc queries through Tendermint.
+* [\#10295](https://github.com/cosmos/cosmos-sdk/pull/10295) Remove store type aliases in /types
```

Comment:
Let's move this to the API-breaking change section

Reply:
@amaurym addressed